eventonica-api/routes/events.js

```diff
@@ -0,0 +1,69 @@
+const express = require('express');
+const cors = require('cors');
+const router = express.Router();
+const Event = require('../models/event');
+const { Op } = require("sequelize");
+
+router.use(express.json());
+router.use(cors());
+router.use(express.urlencoded({ extended: true }))
+
+//GET list of events
+router.get('/', (req, res) => {
+  Event.findAll()
```

Comment:
You could rewrite this with async/await to be consistent with your FE code, e.g.

```
router.get('/', async (req, res) => {
  try {
    const events = await Event.findAll();
    return res.json(events);
  catch (error) {
     console.log('Error:', error);
   }
});
```

I think this should work but there might be some minor gotchas. Would be good to try out.

As another point even if you don't use async/await, I would use an explicit return on line 13. Implicit returns can bite you for anything besides simple one line expressions like line 14.

Reply:
For one thing, I think writing the code in async/await does make it clearer that you aren't specifying what the response should be if there is an error. The server will just hang forever. I actually ran into this trying to run your BE without initializing the DB. This is much harder to debug as the requests to the server just hang as (pending) rather than giving an error (e.g. db connection failed). This would be a good thing for us to fix tomorrow!